qa/workunits/cephtool/test.sh

```diff
@@ -2726,6 +2726,13 @@ function test_osd_compact()
   $SUDO ceph daemon osd.1 compact
 }
 
+function test_osd_daemon_histogram()
+{
+  ceph -h
```

Comment:
```
2021-08-02T07:04:01.065 INFO:tasks.workunit.client.0.smithi043.stdout: Local commands:
2021-08-02T07:04:01.066 INFO:tasks.workunit.client.0.smithi043.stdout: ===============
2021-08-02T07:04:01.066 INFO:tasks.workunit.client.0.smithi043.stdout:
2021-08-02T07:04:01.066 INFO:tasks.workunit.client.0.smithi043.stdout:ping <mon.id>           Send simple presence/life test to a mon
2021-08-02T07:04:01.067 INFO:tasks.workunit.client.0.smithi043.stdout:                        <mon.id> may be 'mon.*' for all mons
2021-08-02T07:04:01.067 INFO:tasks.workunit.client.0.smithi043.stdout:daemon {type.id|path} <cmd>
2021-08-02T07:04:01.067 INFO:tasks.workunit.client.0.smithi043.stdout:                        Same as --admin-daemon, but auto-find admin socket
2021-08-02T07:04:01.067 INFO:tasks.workunit.client.0.smithi043.stdout:daemonperf {type.id | path} [stat-pats] [priority] [<interval>] [<count>]
2021-08-02T07:04:01.068 INFO:tasks.workunit.client.0.smithi043.stdout:daemonperf {type.id | path} list|ls [stat-pats] [priority]
2021-08-02T07:04:01.068 INFO:tasks.workunit.client.0.smithi043.stdout:                        Get selected perf stats from daemon/admin socket
2021-08-02T07:04:01.068 INFO:tasks.workunit.client.0.smithi043.stdout:                        Optional shell-glob comma-delim match string stat-pats
2021-08-02T07:04:01.068 INFO:tasks.workunit.client.0.smithi043.stdout:                        Optional selection priority (can abbreviate name):
2021-08-02T07:04:01.069 INFO:tasks.workunit.client.0.smithi043.stdout:                         critical, interesting, useful, noninteresting, debug
2021-08-02T07:04:01.069 INFO:tasks.workunit.client.0.smithi043.stdout:                        List shows a table of all available stats
2021-08-02T07:04:01.069 INFO:tasks.workunit.client.0.smithi043.stdout:                        Run <count> times (default forever),
2021-08-02T07:04:01.069 INFO:tasks.workunit.client.0.smithi043.stdout:                         once per <interval> seconds (default 1)
2021-08-02T07:04:01.069 INFO:tasks.workunit.client.0.smithi043.stdout:daemonhistogram {daemons} {name} [{mode} [{interval} [{count}]]]
2021-08-02T07:04:01.070 INFO:tasks.workunit.client.0.smithi043.stdout:                        Visualize Ceph daemons' performance histogram.
2021-08-02T07:04:01.070 INFO:tasks.workunit.client.0.smithi043.stdout:                        {daemons} specify where collect data from. This can be
2021-08-02T07:04:01.070 INFO:tasks.workunit.client.0.smithi043.stdout:                          either 'daemon_type.all' specification to use every
2021-08-02T07:04:01.070 INFO:tasks.workunit.client.0.smithi043.stdout:                          daemon of a specific type. Or a comma separated list
2021-08-02T07:04:01.071 INFO:tasks.workunit.client.0.smithi043.stdout:                          of daemon ids or admin socket paths.
2021-08-02T07:04:01.071 INFO:tasks.workunit.client.0.smithi043.stdout:                        {name} is a fully qualified perf histogram name,
2021-08-02T07:04:01.071 INFO:tasks.workunit.client.0.smithi043.stdout:                          full list of names is available through
2021-08-02T07:04:01.071 INFO:tasks.workunit.client.0.smithi043.stdout:                           'ceph daemon <daemon_name> perf histogram schema'
2021-08-02T07:04:01.071 INFO:tasks.workunit.client.0.smithi043.stdout:                        {mode} is either live(default) or batch output mode.
2021-08-02T07:04:01.072 INFO:tasks.workunit.client.0.smithi043.stdout:                        {interval} is a period (in seconds) to update the
2021-08-02T07:04:01.072 INFO:tasks.workunit.client.0.smithi043.stdout:                          histogram, 1 sec if omitted
2021-08-02T07:04:01.072 INFO:tasks.workunit.client.0.smithi043.stdout:                        {count} determines how many times to print,
2021-08-02T07:04:01.072 INFO:tasks.workunit.client.0.smithi043.stdout:                          infinite if omitted
2021-08-02T07:04:01.072 INFO:tasks.workunit.client.0.smithi043.stdout:
```

see https://pulpito.ceph.com/kchai-2021-08-02_06:11:15-rados-wip-kefu-testing-2021-08-02-1223-distro-basic-smithi/6307022/

Reply:
I'm still blaming improper tool version under root for QA failure, Test case fixed and updated - please rerun.